### PR TITLE
added typography and lists

### PR DESCRIPTION
**Pull request by vivek-guda**

### CODE REVIEW

File: Components/typography.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.font-size-xs {
+  font-size: 0.75rem;
+}
+
+.text-center {
+  text-align: center;
+}
+.text-left {
+  text-align: left;
+}
```
Comment (Mohasin-Haque) [inline Components/typography.css:36]: kindly use same formatting. the better way is to run prettier

File: docs.html [hunk 2]
```diff
@@ -67,9 +67,284 @@ <h2 class="nav-title">Style Sheet</h2>
...
+                style="border: 0; transform: scale(1); overflow: hidden"
+                sandbox="allow-scripts allow-same-origin"
+              >
+              </iframe>
+            </code>
+          </div>
+
```
Comment (Mohasin-Haque) [inline docs.html:121]: remove space

File: docs.html [hunk 3]
```diff
@@ -67,9 +67,284 @@ <h2 class="nav-title">Style Sheet</h2>
...
+                style="border: 0; transform: scale(1); overflow: hidden"
```
Comment (Mohasin-Haque) [inline docs.html:115]: avoide inline styling

**Review by Mohasin-Haque — COMMENTED** [review]
Your code is good and clean but needs to work on the comments. Please go through it once.

File: docs.html [hunk 4]
```diff
@@ -67,9 +67,284 @@ <h2 class="nav-title">Style Sheet</h2>
...
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=sharp&l=htmlmixed&width=&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cp%2520class%253D%2522font-size-3xl%2522%253E3XL%2520Font%2520Size%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-2xl%2522%253EXL%2520Font%2520Size%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-xl%2522%253EXL%2520Font%2520Size%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-lg%2522%253ELG%2520Font%2520Size%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-md%2522%253EMD%2520Font%2520Size%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-rg%2522%253ERG%2520Font%2520Size%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-sm%2522%253ESM%2520Font%2520Size%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-xs%2522%253EXS%2520Font%2520Size%253C%252Fp%253E"
+                style="border: 0; transform: scale(1); overflow: hidden"
+                sandbox="allow-scripts allow-same-origin"
+              >
+              </iframe>
+            </code>
+          </div>
+
```
Comment (arjunbharti) [inline docs.html:272]: Remove extra spacing

File: styles.css [hunk 5]
```diff
@@ -18,20 +18,25 @@
   --footer-color: #ddd;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
 body {
   background-color: var(--background-color);
   color: #3e4152;
```
Comment (arjunbharti) [inline styles.css:27]: use color variable

File: styles.css [hunk 6]
```diff
@@ -18,20 +18,25 @@
...
 }
 
-li {
-  list-style: none;
-}
-
 a {
   text-decoration: none;
   color: #000;
```
Comment (arjunbharti) [inline styles.css:32]: use color variable

**Review by arjunbharti — COMMENTED** [review]
Most of the things are good. Just added some suggestions.

**Pull request merged**